CONTRIBUTING.md

```diff
@@ -8,6 +8,12 @@ for a friendly and welcoming collaborative environment.
 
 ## Setting up a development environment
 
+<!--
```

Comment:
Feel free to add a link here to the Documentation hosted on RTD. I could see this `CONTRIBUTING.md` being minimal docs targeted for experienced developers where the more thorough docs could be on RTD.